docs/competition/team-support/battery-charging.md

```diff
@@ -7,44 +7,76 @@ original:
 
 We (Student Robotics) manage the charging of all batteries at the competition. Battery Charging volunteers are responsible for ensuring that there is a continuous supply of charged batteries available for use by teams.
 
-## Equipment
+## Setup
 
- * Battery chargers
- * Discharged battery box
- * Charged battery box
- * Damaged battery/charger box
- * Red insulating tape
- * Example good batteries (one brand new, one squidgy)
+### Equipment
+
+There are two locations involved in battery charging:
+
+- Helpdesk
+    - 1 x Discharged Battery Box (18L RUB), clearly labelled
+    - 1 x Charged Battery Box (18L RUB), clearly labelled
+    - 1 x LiPo Bag for damaged batteries, clearly labelled, red taped
+    - At least one roll of red insulating tape
+- Battery Charging Station
+    - At least `n` battery chargers
+    - Enough LiPo charging bags for every charger.
```

Comment:
I don't think these two requirements make sense -- the majority of our chargers and bags are with teams and are only collected when they arrive. As a result, the charging station cannot have them set up until the teams arrive.

Reply:
I will remove these for the next PR. Is this list otherwise fine?

Reply:
I think so. Maybe note here explicitly that the chargers and bags will arrive when the teams arrive, though we should try to set up as many as we can from our other stock?

For this year (SR2019), I think we'll likely be able to fully set up ahead of time as we have many more chargers than teams, though I don't expect that to be the usual case.